connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/config/EmailConnectorConfig.java

```diff
@@ -18,11 +24,17 @@ public class EmailConnectorConfig extends ConnectorConfig {
     private static final String IT_KEYSTORE_LOCATION = "notifications.connector.user-provider.it.key-store-location";
     private static final String IT_KEYSTORE_PASSWORD = "notifications.connector.user-provider.it.key-store-password";
     private static final String IT_USER_SERVICE_URL = "notifications.connector.user-provider.it.url";
+    private static final String RBAC_APPLICATION_KEY = "notifications.connector.user-provider.rbac.application-key";
     private static final String RBAC_ELEMENTS_PAGE = "notifications.connector.user-provider.rbac.elements-per-page";
     private static final String RBAC_URL = "notifications.connector.user-provider.rbac.url";
     @Deprecated(forRemoval = true)
     private static final String SINGLE_EMAIL_PER_USER = "notifications.connector.single-email-per-user.enabled";
 
+    // The following two keys are public in order to make it easier for
+    // overriding them in the tests.
+    public static final String RBAC_DEVELOPMENT_AUTHENTICATION_KEY = "notifications.connector.user-provider.rbac.development-authentication-key";
+    public static final String RBAC_PSKS = "notifications.connector.user-provider.rbac.psks";
```

Comment:
Does the ClowdApp template need to be updated?

Reply:
Yes, done!